src/app/home.js

```diff
@@ -46,7 +46,14 @@ const MenuItem = ({item}) => r(
     item.submenu && [SubMenu, {items: item.submenu}],
   ])
 
-export default (menu) => r(
+const Text = ({textWithLink}) => r(
```

Comment:
I would name the component `TextWithLink` and supply two properties, text and link instead of wrapping them into a `textWithLink` property